src/Game.java

```diff
@@ -4,95 +4,94 @@
 import java.util.Random;
 
 public class Game {
-    boolean whose_turn;
-    public boolean is_running;
+    boolean whoseTurn;
+    boolean isRunning;
     boolean draw;
     int[][] fields = new int[3][3];
-    int type_of_win = 0;
-    boolean number_of_players;
+    int typeOfWin = 0;
+    boolean numberOfPlayers;
     Game(){
-        is_running = true;
+        isRunning = true;
         draw = false;
-        whose_turn = pick_who_starts();
+        whoseTurn = pickWhoStarts();
         for(int i=0; i<fields.length; i++){
             for(int j=0; j<fields[0].length; j++){
                 fields[i][j] = 0;
             }
         }
     }
-    public boolean pick_who_starts(){
+    public boolean pickWhoStarts(){
         Random random = new Random();
         return random.nextBoolean();
     }
-    public int check_winner(){
+    public int checkWinner(){
```

Comment:
Może warto byłoby zwracać tutaj jakiś enum z wartościami X / O / Draw? byłoby to bardziej czytelne niż zwracać wartość, a tutaj mamy tylko 3 możliwe opcje wyniku gry

Reply:
Jak wyżej, zmieniłam tą metodę, więc komentarz jest chyba nieaktualny.